pkg/util/file.go

```diff
@@ -48,3 +48,29 @@ func WriteFileToLog(logger *capnslog.PackageLogger, path string) {
 
 	logger.Infof("Config file %s:\n%s", path, string(contents))
 }
+
+func FileExists(path string) (bool, error) {
+	_, err := os.Stat(path)
+	if err == nil {
+		return true, nil
+	}
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+	return true, err
+}
+
+func StringReplace(file, originalString, replaceWith string, occurrences int) error {
+	input, err := ioutil.ReadFile(file)
+	if err != nil {
+		return fmt.Errorf("failed to update file. %+v", err)
+	}
+
+	output := bytes.Replace(input, []byte(originalString), []byte(replaceWith), occurrences)
+
+	if err = ioutil.WriteFile(file, output, 0666); err != nil {
```

Comment:
`0644` should be sufficient for you to write it

Reply:
updated